index.d.ts

```diff
@@ -3148,7 +3148,7 @@ declare namespace Eris {
     deny: bigint;
     json: Record<keyof Constants["Permissions"], boolean>;
     constructor(allow: number | string | bigint, deny?: number | string | bigint);
-    has(permission: keyof Constants["Permissions"]): boolean;
+    has(permission: keyof Constants["Permissions"] | BigInt): boolean;
```

Comment:
```suggestion
    has(permission: keyof Constants["Permissions"] | bigint): boolean;
```
Prefer using primitive type instead of class